lib/assets/javascripts/dmproadmap/forms.js

```diff
@@ -59,7 +59,7 @@ function validatePassword(sPassword) {
   if(sPassword.trim().length >= 8 && sPassword.trim().length <= 128){
     return '';
   }else{
-    return __('Passwords must have at least 8 characters');
+    return 'Passwords must have at least 8 characters';
```

Comment:
I guess is hardcoded since this file is conforming to ES5 still.

Reply:
Yes, it can't find `__` anymore and I knew we were moving to a different solution for translations so figured I would just hard code the value for now